codegen/smithy-aws-go-codegen/build.gradle.kts

```diff
@@ -34,6 +34,7 @@ tasks.withType<Test> {
 }
 
 dependencies {
+	implementation("software.amazon.smithy:smithy-cli:$smithyVersion")
```

Comment:
This should only be in sdk-codegen and protocol-test-codegen as its the project that pull in the CL via the plugin that require the fix.

Reply:
Fixed by putting the `smithy-cli` in a `buildScript` section in the correct projects.